archivebox/main.py

```diff
@@ -597,7 +578,7 @@ def remove(filter_str: Optional[str]=None,
     
     check_data_folder(out_dir=out_dir)
 
-    if links is None:
+    if not snapshots:
```

Comment:
```suggestion
    if snapshots is None:
```

An empty QuerySet is still a valid QuerySet, only `None` is invalid.

Reply:
https://github.com/pirate/ArchiveBox/pull/452/commits/45e6bf555c608325e571104d05e7e778640b2578